nx-X11/programs/Xserver/randr/rrdispatch.c

```diff
@@ -21,48 +21,62 @@
  */
 
 #include "randrstr.h"
-
-#define SERVER_RANDR_MAJOR	1
-#define SERVER_RANDR_MINOR	2
+#ifndef NXAGENT_SERVER
```

Comment:
I think it would be better to not abuse NXAGENT_SERVER for stuff that is missing in the used X11 base. Maybe these places should be changed to XORG_NOT_YET_EXISTING or something...